opm/parser/eclipse/EclipseState/TracerConfig.hpp

```diff
@@ -34,7 +34,9 @@ class TracerConfig {
         std::string name;
         Phase phase = Phase::OIL;
         std::vector<double> concentration;
+        std::vector<double> concentrationS;
```

Comment:
What is the capital S?

Reply:
Oil tracers when VAPOIL is active, require specification also of tracer initially in solution (S).  Similarly for gas tracers and DISGAS. Maybe F (free) should be added to `concentration` similar to what is done for `tvdpf`?

Reply:
OK - how about `free_concentration` or `solution_concentration`? The way it is now it looked like a a capitalization.